src/overlays/actors/ovl_En_Yabusame_Mark/z_en_yabusame_mark.c

```diff
@@ -40,17 +29,208 @@ static ColliderQuadInit D_80B434F0 = {
         ELEMTYPE_UNK0,
         { 0x00000000, 0x00, 0x00 },
         { 0x0001F824, 0x00, 0x00 },
-        TOUCH_NONE,
+        TOUCH_NONE | TOUCH_SFX_NORMAL,
         BUMP_ON,
         OCELEM_NONE,
     },
     { { { 0.0f, 0.0f, 0.0f }, { 0.0f, 0.0f, 0.0f }, { 0.0f, 0.0f, 0.0f }, { 0.0f, 0.0f, 0.0f } } },
 };
-*/
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Yabusame_Mark/EnYabusameMark_Destroy.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Yabusame_Mark/EnYabusameMark_Init.s")
+const ActorInit En_Yabusame_Mark_InitVars = {
+    ACTOR_EN_YABUSAME_MARK,
+    ACTORCAT_PROP,
+    FLAGS,
+    OBJECT_GAMEPLAY_KEEP,
+    sizeof(EnYabusameMark),
+    (ActorFunc)EnYabusameMark_Init,
+    (ActorFunc)EnYabusameMark_Destroy,
+    (ActorFunc)EnYabusameMark_Update,
+    NULL,
+};
 
+static Vec3f collisionVerticies[] = {
+    { 70.0f, 70.0f, 0.0f },      { 70.0f, -70.0f, 0.0f },      { -70.0f, 70.0f, 0.0f },    { -70.0f, -70.0f, 0.0f },
```

Comment:
like roman mentioned in the other comment, this needs an `s` prefix then its good to go